scripts/stack/connectors-config/config.yml

```diff
@@ -0,0 +1,226 @@
+## ================= Elastic Connectors Configuration ==================
```

Comment:
I personally really dislike that now we have 3 config files in the repository - is there a way to avoid this?

Additionally, we don't follow the `config.yml.example` pattern with our regular config, why have one here?

Reply:
Ah - good point. Since I have it set for the Connectors-CLI to modify the config file in place, I copied my own version of the config... _but_ I can check and copy the regular config and not keep my own... thanks for this.